pydov/search/interpretaties.py

```diff
@@ -7,6 +7,7 @@
 from pydov.types.interpretaties import LithologischeBeschrijvingen
 from pydov.types.interpretaties import GecodeerdeLithologie
 from pydov.types.interpretaties import GeotechnischeCodering
+from pydov.types.interpretaties import QuartaireStratigrafie
```

Comment:
Actually 'Quartairstratigrafie' is the better term. I would suggest changing it everywhere (except the WFS layer name) from 'quartaire stratigrafie' to 'quartairstratigrafie'.

```suggestion
from pydov.types.interpretaties import QuartairStratigrafie
```